test/DebugInfo/async-lifetime-extension.swift

```diff
@@ -10,14 +10,14 @@
 // CHECK-NEXT: entryresume.0:
 // CHECK-NEXT: call void @llvm.dbg.declare(metadata {{.*}}%0, metadata ![[RHS:[0-9]+]], {{.*}}!DIExpression(DW_OP
 // CHECK-NEXT: call void @llvm.dbg.declare(metadata {{.*}}%0, metadata ![[LHS:[0-9]+]], {{.*}}!DIExpression(DW_OP
-// CHECK-NEXT: call void @llvm.dbg.declare(metadata {{.*}}%0, metadata ![[N:[0-9]+]], {{.*}}!DIExpression(DW_OP
 // CHECK-NEXT: call void @llvm.dbg.declare(metadata {{.*}}%0, metadata ![[R:[0-9]+]], {{.*}}!DIExpression(DW_OP
+// CHECK-NEXT: call void @llvm.dbg.declare(metadata {{.*}}%0, metadata ![[N:[0-9]+]], {{.*}}!DIExpression(DW_OP
 // CHECK-NOT: {{ ret }}
 // CHECK: call void asm sideeffect ""
 // CHECK: ![[RHS]] = !DILocalVariable(name: "rhs"
 // CHECK: ![[LHS]] = !DILocalVariable(name: "lhs"
-// CHECK: ![[N]] = !DILocalVariable(name: "n"
 // CHECK: ![[R]] = !DILocalVariable(name: "retval"
+// CHECK: ![[N]] = !DILocalVariable(name: "n"
```

Comment:
I think CHECK-DAG can alleviate some of this churn, but perhaps that's a change for later.

Reply:
Correct —  it's also considerably slower, so I try to avoid it if for mostly stable output.